ui/selectors/selectors.js

```diff
@@ -826,6 +826,7 @@ function getAllowedAnnouncementIds(state) {
     10: Boolean(process.env.TOKEN_DETECTION_V2) && !process.env.IN_TEST,
     11: Boolean(process.env.TOKEN_DETECTION_V2) && !process.env.IN_TEST,
     12: true,
+    13: Boolean(process.env.ADD_POPULAR_NETWORKS) && !process.env.IN_TEST,
```

Comment:
```suggestion
    13: Boolean(process.env.ADD_POPULAR_NETWORKS),
```
`!process.env.IN_TEST` can be removed by adding `13: {"isShown": true}` in the NotificatioonController object in all `state.json` files; it will help reduce the work during flag removal.

Reply:
Done!